packages/network-controller/tests/provider-api-tests/no-block-param.ts

```diff
@@ -77,7 +65,8 @@ export const testsForRpcMethodAssumingNoBlockParam = (
   });
 
   for (const paramIndex of [...Array(numberOfParameters).keys()]) {
-    it(`does not reuse the result of a previous request if parameter at index "${paramIndex}" differs`, async () => {// eslint-disable-line
+    // eslint-disable-next-line no-loop-func
```

Comment:
What's this (and other lines like it) for?

Reply:
just a mre specific version of what was already there

Reply:
Oh, our ESLint config doesn't think these are Jest files, gotcha.